front/lib/user.ts

```diff
@@ -58,3 +58,18 @@ export function setUserMetadataFromClient(metadata: UserMetadataType) {
     }
   })();
 }
+
+export const guessFirstandLastNameFromFullName = (
+  fullName: string
+): { firstName: string | null; lastName: string | null } => {
+  if (!fullName) return { firstName: null, lastName: null };
+
+  const nameParts = fullName.split(" ");
+
+  if (nameParts.length === 1) return { firstName: fullName, lastName: null };
+
+  const lastName = nameParts.pop() || null;
```

Comment:
We should put all parts in last name not first name? no?

Reply:
Rationale was I expected most people with 3 name parts were a composed first name and a last name, but it was not a strong conviction, I'll make the change 👍🏻